Installation issue: `paraview~mpi`: "An external MPI-aware HDF5 requires that VTK be built with MPI support as well"

### Steps to reproduce the issue

Reproduces on two `spack` trees on two machines:
```console
$ spack install paraview~development_files~kits~mpi~shared~cuda~adios2+fides

.stage/spack-stage-paraview-5.13.1-ami44zd4mpzuklsdj2za5ln6oe4eguux/spack-configure-args.txt is:

-DVTK_OPENGL_HAS_OSMESA:BOOL=OFF -DVTK_USE_X:BOOL=ON -DPARAVIEW_INSTALL_DEVELOPMENT_FILES:BOOL=OFF -DBUILD_TESTING:BOOL=OFF -DOpenGL_GL_PREFERENCE:STRING=LEGACY -DPARAVIEW_ENABLE_VISITBRIDGE:BOOL=OFF -DVISIT_BUILD_READER_Silo:BOOL=OFF -DVTK_MODULE_USE_EXTERNAL_VTK_fast_float:BOOL=OFF -DVTK_MODULE_USE_EXTERNAL_VTK_token:BOOL=OFF -DVTK_MODULE_USE_EXTERNAL_VTK_verdict:BOOL=OFF -DVTK_MODULE_USE_EXTERNAL_ParaView_vtkcatalyst:BOOL=OFF -DVTK_MODULE_USE_EXTERNAL_VTK_ioss:BOOL=OFF -DVTK_MODULE_USE_EXTERNAL_VTK_exprtk:BOOL=OFF -DVTK_MODULE_USE_EXTERNAL_VTK_fmt:BOOL=OFF -DPARAVIEW_BUILD_EDITION:STRING=CANONICAL -DPARAVIEW_USE_QT:BOOL=OFF -DPARAVIEW_BUILD_WITH_EXTERNAL=ON -DPARAVIEW_ENABLE_EXAMPLES:BOOL=OFF -DVTK_MODULE_USE_EXTERNAL_ParaView_cgns=OFF -DVTK_MODULE_USE_EXTERNAL_VTK_gl2ps=OFF -DVTK_MODULE_USE_EXTERNAL_VTK_libharu=OFF -DVTK_MODULE_USE_EXTERNAL_VTK_utf8=OFF -DPARAVIEW_ENABLE_PYTHON:BOOL=OFF -DPARAVIEW_BUILD_SHARED_LIBS:BOOL=OFF -DPARAVIEW_USE_CUDA:BOOL=OFF -DPARAVIEW_USE_HIP:BOOL=OFF -DPARAVIEW_ENABLE_RAYTRACING:BOOL=OFF -DVTK_ENABLE_OSPRAY:BOOL=OFF -DVTKOSPRAY_ENABLE_DENOISER:BOOL=OFF -DPARAVIEW_PLUGIN_ENABLE_CDIReader:BOOL=OFF -DPARAVIEW_PLUGIN_AUTOLOAD_CDIReader:BOOL=OFF

...
```


### Error message

```py
-- Found HDF5: hdf5-shared (found version "1.14.5") found components: C HL
CMake Error at VTK/ThirdParty/hdf5/CMakeLists.txt:31 (message):
  An external MPI-aware HDF5 requires that VTK be built with MPI support as
  well.
```

### Information on your system

On PR branch of #46971:

bin/spack debug report
* **Spack:** 0.23.0.dev0 (d14432282d15d6fca70c780e96b65f8db163c98e)
* **Python:** 3.10.12
* **Platform:** linux-ubuntu22.04-skylake

### Additional information

```py
==> Installing paraview-5.13.1-ami44zd4mpzuklsdj2za5ln6oe4eguux [87/87]
==> No binary for paraview-5.13.1-ami44zd4mpzuklsdj2za5ln6oe4eguux found: installing from source
==> Fetching http://www.paraview.org/files/v5.13/ParaView-v5.13.1.tar.xz
==> Applied patch var/spack/repos/builtin/packages/paraview/FindFreetype.cmake.patch
==> paraview: Executing phase: 'cmake'
==> [2024-10-19-13:03:56.589772] 'opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/cmake-3.30.5-h23fmzvmea77a4jg3dihcy4bz72jkk6b/bin/cmake' '-G' 'Ninja' -DCMAKE_PREFIX_PATH:STRING=opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/pugixml-1.10-2r72qtte56iwzvlnuqo4wigxpd5ofe64;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/protobuf-3.13.0-6p4hstndxpeeehliu53lksrkz4sgxjlp;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/proj-8.1.0-hx3d5v2h2zvgdvahke7vt3i33qbuz5bz;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/pegtl-2.8.3-5u5wvepk6t7fekt22fzkdf6gdbavecoa;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/nlohmann-json-3.11.3-ubipdqjwm3qag76vzl7n5wzv2x345fmh;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/netcdf-c-4.9.2-6euhkr4keva27azw2dxppn4rm7oix5d3;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libaec-1.0.6-gkzdtmodrwag7p4awsix33ekbiywffsr;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/hdf5-1.14.5-bxuqkul72wmaxuzdy36tvjk2ktqyciuq;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/openmpi-5.0.5-ykvt4t64hyztqgujekybpjnl4nfd6as5;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/pmix-5.0.3-4fdv6pbuworf2hsnkwp2bnfkwtemfmh5;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/numactl-2.0.18-ot33hns6raosys5gi2l5hcdvqk372vwq;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libevent-2.1.12-s2ur47rbs6aygom4to6grygx4uxy5ats;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/c-blosc-1.21.5-hvaklx4p3nnqpng45rzbrwrbna7igkv2;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/snappy-1.1.10-fmwupli2sikp3jfbedobhp4a3iwtr4te;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/lz4-1.10.0-jjeqt2kye2avym36cxlke46s64zsfs6v;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libtiff-4.6.0-mdg3npn5i7hfzf5cqqywaunjzd2c46zu;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libtheora-1.1.1-joyidf5le4lbyztj36exsuebtjewsfbd;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libogg-1.3.5-66xpqstxcbgw2ncqgx2vzmqsuwtldugx;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libjpeg-turbo-3.0.3-tffgl7ijaojo5ec4ojr6lniafj7zqkfx;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/jsoncpp-1.9.5-467aebg2dqecutvtmsjz73fgjvzydl6y;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/glew-2.2.0-mo5jcq3naak3y6sdt5ltzkdhcpi4g4jz;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/glx-1.4-ldb6nnyqutqwoj4snm2ia7f7aokvcbp2;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/mesa-23.3.6-ccdk6jiz6lhbdfkyrapqqwwwzexcl2ks;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/xrandr-1.5.2-vottpkk73ljuar3hpvjqfalhlsvktejg;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxrandr-1.5.4-4eadn67orgvcgjuzrettm6gcg67tu2ux;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/randrproto-1.5.0-tlbxpvcglzmzmnow55rcgicl4bierdlu;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxrender-0.9.11-6lvxhpcdewfs4mafcm5j7kg4fpcxa7j7;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/renderproto-0.11.1-od2tcw3fbz2iu4pi7o6bnv4avuiyfxld;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/llvm-17.0.6-hywrlwv2bskuoca7fe6lfohwpctojfpf;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/swig-4.1.1-pbo27h3jn2t62tkxzwko2i4psb5ohnfs;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/pcre2-10.44-ohomucklfkbvybtvne7ui3glenebe7fx;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/ninja-1.12.1-wepbmyred3fgz62zbysduq6rxb4g646x;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/lua-5.3.6-xialgefhbgjzqhdhwdsuogpysjnlqm3q;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libedit-3.1-20240808-upfwxlljipi45okiyjpcxjbimzcn3tqb;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/hwloc-2.11.1-znd5x5ytxhvjepa3k2ordikjxk56ghdp;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libpciaccess-0.17-azx6duzhod5abnalp7keendpz4huhru4;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/binutils-2.43.1-4sodjgf6cc7uekitqepcc7dnmcwtaho3;/home/bkaindl/spack.5/opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxt-1.3.0-ocjvjxavofrzwbihg4bpsqpagc3ofb2w;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libsm-1.2.4-o6cqylf6wpq5lxt664h555u2aqmv2gl7;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libice-1.1.1-4zsfoqozxzxie5grnkxwxr3onssb5ezk;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxext-1.3.6-frmdae6dcynu4535k2ypww5jggnwpfxz;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libx11-1.8.10-dtf7uji6hrpgtmnjfljpwtx4bmdel3hm;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/xtrans-1.5.0-k5kj5vhxaeeqafuhcwohtww7fduyfqmr;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/xextproto-7.3.0-vwtxuuuxps6dvrpf7cv7h7iurz76cquh;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxcb-1.17.0-rn4kx2ryk5qk5qqxey4qq6u6db7mfprs;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/util-linux-uuid-2.40.2-tyzqiklai55oisaoewn5ro5nv3wy7mcv;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/sqlite-3.46.0-i6xusas3hfs57j5p72at6gucksqd3gsw;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libffi-3.4.6-r6emmhbggetnzptutbkk7qffbsmezppz;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxdmcp-1.1.5-xxkwiljmjja7w7sp3kynvqyjz33rdiog;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxau-1.0.11-ldkt5yi3sok6eqydtuqzy6ubl2gk3bc4;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/xproto-7.0.31-xddze4r3uhsr6h27xijkzckrikpxoznf;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libpthread-stubs-0.5-doadflxu2oxvbyahsxtvvbfnw6tnn6z4;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/kbproto-1.0.7-jq43mwjkv35hsuxwa3rq7ahjncosez3u;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libunwind-1.6.2-u5yu7dhud7vpj3hyoty6x6357b7o3oif;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/glproto-1.4.17-dqs2nfuwx3ioojm27hnqbn7su4sgirll;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/zstd-1.5.6-3oda7hrcm25fqymbhvrblqwlmpj7hwqh;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxml2-2.10.3-2cbjv2cwl4mm5bxmrwhiif6ghxrsf75w;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/xz-5.4.6-pwjjfgxqlhfn26tycgwy2iezdpzmt7br;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/freetype-2.13.3-hbq2y76onlaasodnlyex5iv745hhwpwl;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libpng-1.6.39-6ewrjni6gx3fuxbx7gy5b3giz7wtpwrx;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/expat-2.6.3-u2bucaey36mxkx6zbov2tk46okm3zsh2;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libbsd-0.12.2-calffvojasv62v5v6b4mplsycwbj67fs;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libmd-1.0.4-z4llf26xjzstptwwlm4efpzljny4hb47;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/eigen-3.4.0-bdamxaxlisqoxb2fuz4pb4vsxvx56gh5;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/double-conversion-3.3.0-4iuaeml6dsfeduksvrph3kaqgmcjxk5i;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/cli11-1.9.1-5r37fzco3pn3mdllq5a2swwiacxahl5v;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/cmake-3.30.5-h23fmzvmea77a4jg3dihcy4bz72jkk6b;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/curl-8.8.0-onnikjqv4vrfb4peuleyceuqwtv5sild;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/openssl-3.3.1-ry7k4rpxiacirqdesthhlc2c6mh5hhgy;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/zlib-ng-2.2.1-somd2q767lkqx6je2pz6am2gztdbovfe;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/readline-8.2-w6hn7avdihv3xjsswtwougplip4ulmgu;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/ncurses-6.5-5iauhqykhluyoehuhrwhvvuyvpud2765;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/nghttp2-1.63.0-kswzwok6iadc6xko2iwxljpij5y2gzd3;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/bzip2-1.0.8-geajalrno4mt3uers3ji7ytbkk6qvkez;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libiconv-1.17-jxe63kwxqplstnvcueiggoke5v2ezi6d;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/gcc-runtime-13.1.0-25qo3ld7klulv5juopmjnk73qte743xz' -DVTK_MODULE_USE_EXTERNAL_VTK_fast_float:BOOL=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_token:BOOL=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_verdict:BOOL=OFF' '-DVTK_MODULE_USE_EXTERNAL_ParaView_vtkcatalyst:BOOL=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_ioss:BOOL=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_exprtk:BOOL=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_fmt:BOOL=OFF' '-DPARAVIEW_BUILD_EDITION:STRING=CANONICAL' '-DPARAVIEW_USE_QT:BOOL=OFF' '-DPARAVIEW_BUILD_WITH_EXTERNAL=ON' '-DPARAVIEW_ENABLE_EXAMPLES:BOOL=OFF' '-DVTK_MODULE_USE_EXTERNAL_ParaView_cgns=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_gl2ps=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_libharu=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_utf8=OFF' '-DPARAVIEW_ENABLE_PYTHON:BOOL=OFF' '-DPARAVIEW_BUILD_SHARED_LIBS:BOOL=OFF' '-DPARAVIEW_USE_CUDA:BOOL=OFF' '-DPARAVIEW_USE_HIP:BOOL=OFF' '-DPARAVIEW_ENABLE_RAYTRACING:BOOL=OFF' '-DVTK_ENABLE_OSPRAY:BOOL=OFF' '-DVTKOSPRAY_ENABLE_DENOISER:BOOL=OFF' '-DPARAVIEW_PLUGIN_ENABLE_CDIReader:BOOL=OFF' '-DPARAVIEW_PLUGIN_AUTOLOAD_CDIReader:BOOL=OFF' '/home/bkaindl/spack.5/.stage/spack-stage-paraview-5.13.1-ami44zd4mpzuklsdj2za5ln6oe4eguux/spack-src'
-- The C compiler identification is GNU 13.1.0
-- The CXX compiler identification is GNU 13.1.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: lib/spack/env/gcc/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: lib/spack/env/gcc/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test HAVE_GCC_ERROR_RETURN_TYPE
-- Performing Test HAVE_GCC_ERROR_RETURN_TYPE - Success
-- Looking for a Fortran compiler
-- Looking for a Fortran compiler - lib/spack/env/gcc/gfortran
CMake Warning at CMake/ParaViewSupportMacros.cmake:25 (message):
  The 'PARAVIEW_ENABLE_PYTHON' variable is deprecated for
  'PARAVIEW_USE_PYTHON'.
Call Stack (most recent call first):
  CMake/ParaViewOptions.cmake:131 (vtk_deprecated_setting)
  CMakeLists.txt:104 (include)


CMake Warning (dev) at CMake/ParaViewMigrations.cmake:7 (message):
  The `VTK_MODULE_USE_EXTERNAL_ParaView_cgns` setting has been migrated to
  `VTK_MODULE_USE_EXTERNAL_VTK_cgns`.
Call Stack (most recent call first):
  CMake/ParaViewMigrations.cmake:18 (paraview_migrate_setting)
  CMakeLists.txt:105 (include)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Performing Test CMAKE_REQUIRE_LARGE_FILE_SUPPORT
-- Performing Test CMAKE_REQUIRE_LARGE_FILE_SUPPORT - Success
-- Could not use git to determine source version, using version 5.13.1
-- The Fortran compiler identification is GNU 13.1.0
-- Detecting Fortran compiler ABI info
-- Detecting Fortran compiler ABI info - done
-- Check for working Fortran compiler: lib/spack/env/gcc/gfortran - skipped
-- Could not use git to determine source version, using version
-- VTK version: 9.3.20240617
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY - Success
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY - Success
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Success
-- Checking whether wstring is available
-- Checking whether wstring is available - yes
-- Checking whether C compiler has ptrdiff_t in stddef.h
-- Checking whether C compiler has ptrdiff_t in stddef.h - yes
-- Checking whether C compiler has ssize_t in unistd.h
-- Checking whether C compiler has ssize_t in unistd.h - yes
-- Checking whether C compiler has clock_gettime
-- Checking whether C compiler has clock_gettime - yes
-- Checking whether CXX compiler has setenv
-- Checking whether CXX compiler has setenv - yes
-- Checking whether CXX compiler has unsetenv
-- Checking whether CXX compiler has unsetenv - yes
-- Checking whether CXX compiler has environ in stdlib.h
-- Checking whether CXX compiler has environ in stdlib.h - no
-- Checking whether CXX compiler has utimes
-- Checking whether CXX compiler has utimes - yes
-- Checking whether CXX compiler has utimensat
-- Checking whether CXX compiler has utimensat - yes
-- Checking whether CXX compiler struct stat has st_mtim member
-- Checking whether CXX compiler struct stat has st_mtim member - yes
-- Checking whether CXX compiler struct stat has st_mtimespec member
-- Checking whether CXX compiler struct stat has st_mtimespec member - no
-- Looking for include files sys/types.h, ifaddrs.h
-- Looking for include files sys/types.h, ifaddrs.h - found
-- Checking whether CXX compiler has rlimit64
-- Checking whether CXX compiler has rlimit64 - yes
-- Looking for C++ include execinfo.h
-- Looking for C++ include execinfo.h - found
-- Checking whether backtrace works with this C++ compiler
-- Checking whether backtrace works with this C++ compiler - yes
-- Looking for C++ include dlfcn.h
-- Looking for C++ include dlfcn.h - found
-- Checking whether dladdr works with this C++ compiler
-- Checking whether dladdr works with this C++ compiler - yes
-- Looking for C++ include cxxabi.h
-- Looking for C++ include cxxabi.h - found
-- Checking whether cxxabi works with this C++ compiler
-- Checking whether cxxabi works with this C++ compiler - yes
-- Checking whether CXX compiler has getloadavg
-- Checking whether CXX compiler has getloadavg - yes
-- Checking whether <ext/stdio_filebuf.h> is available
-- Checking whether <ext/stdio_filebuf.h> is available - yes
-- Found nlohmann_json: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/nlohmann-json-3.11.3-ubipdqjwm3qag76vzl7n5wzv2x345fmh/share/cmake/nlohmann_json/nlohmann_jsonConfig.cmake (found version "3.11.3")
-- Looking for C++ include cxxabi.h
-- Looking for C++ include cxxabi.h - found
-- Looking for abi:__cxa_demangle
-- Looking for abi:__cxa_demangle - found
-- Found double-conversion: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/double-conversion-3.3.0-4iuaeml6dsfeduksvrph3kaqgmcjxk5i/lib/libdouble-conversion.a
-- Looking for <cxxabi.h>
-- Looking for <cxxabi.h> - not found
-- Looking for <dlfcn.h>
-- Looking for <dlfcn.h> - not found
-- Looking for <execinfo.h>
-- Looking for <execinfo.h> - not found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
CMake Warning (dev) at opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/cmake-3.30.5-h23fmzvmea77a4jg3dihcy4bz72jkk6b/share/cmake-3.30/Modules/CMakeDependentOption.cmake:89 (message):
  Policy CMP0127 is not set: cmake_dependent_option() supports full Condition
  Syntax.  Run "cmake --help-policy CMP0127" for policy details.  Use the
  cmake_policy command to set the policy and suppress this warning.
Call Stack (most recent call first):
  VTK/Common/Core/CMakeLists.txt:79 (cmake_dependent_option)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Checking for builtin __atomic_add_fetch
-- Checking for builtin __atomic_add_fetch -- success
-- Performing Test VTK_HAS_STD_ISNAN
-- Performing Test VTK_HAS_STD_ISNAN - Success
-- Performing Test VTK_HAS_STD_ISINF
-- Performing Test VTK_HAS_STD_ISINF - Success
-- Performing Test VTK_HAS_STD_ISFINITE
-- Performing Test VTK_HAS_STD_ISFINITE - Success
-- Looking for isnan
-- Looking for isnan - found
-- Looking for isinf
-- Looking for isinf - found
-- Looking for isfinite
-- Looking for isfinite - found
-- Looking for finite
-- Looking for finite - found
-- Looking for fenv.h
-- Looking for fenv.h - found
-- Looking for feenableexcept
-- Looking for feenableexcept - found
-- Looking for abi:__cxa_demangle
-- Looking for abi:__cxa_demangle - found
-- Looking for getsockname in socket
-- Looking for getsockname in socket - not found
-- Checking for getsockname with socklen_t
-- Checking for getsockname with socklen_t -- 1
-- Looking for SO_REUSEADDR
-- Looking for SO_REUSEADDR - found
-- Found PEGTL: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/pegtl-2.8.3-5u5wvepk6t7fekt22fzkdf6gdbavecoa/include/tao (found suitable version "2.8.3", minimum required is "2.0.0")
-- Looking for strtod_l
-- Looking for strtod_l - not found
-- Found ZLIB: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/zlib-ng-2.2.1-somd2q767lkqx6je2pz6am2gztdbovfe/lib/libz.so (found version "1.3.1")
-- Could NOT find freetype (missing: freetype_DIR)
-- Could NOT find freetype (missing: freetype_DIR)
-- Found Freetype: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/freetype-2.13.3-hbq2y76onlaasodnlyex5iv745hhwpwl/lib/libfreetype.so (found version "2.13.3")
-- Found JPEG: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libjpeg-turbo-3.0.3-tffgl7ijaojo5ec4ojr6lniafj7zqkfx/lib/libjpeg.so (found version "62")
-- Found PNG: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libpng-1.6.39-6ewrjni6gx3fuxbx7gy5b3giz7wtpwrx/lib/libpng.so (found version "1.6.39")
-- Found TIFF: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libtiff-4.6.0-mdg3npn5i7hfzf5cqqywaunjzd2c46zu/lib/cmake/tiff (found version "4.6.0")
-- Found Eigen3: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/eigen-3.4.0-bdamxaxlisqoxb2fuz4pb4vsxvx56gh5/include/eigen3 (found version "3.4.0")
-- Found LZ4: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/lz4-1.10.0-jjeqt2kye2avym36cxlke46s64zsfs6v/lib/liblz4.so (found version "1.10.0")
-- Found LZMA: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/xz-5.4.6-pwjjfgxqlhfn26tycgwy2iezdpzmt7br/lib/liblzma.so (found version "5.4.6")
-- Found EXPAT: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/expat-2.6.3-u2bucaey36mxkx6zbov2tk46okm3zsh2/lib/libexpat.so (found version "2.6.3")
-- Found OpenGL: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/mesa-23.3.6-ccdk6jiz6lhbdfkyrapqqwwwzexcl2ks/lib/libGL.so  found components: OpenGL
-- Found X11: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/xproto-7.0.31-xddze4r3uhsr6h27xijkzckrikpxoznf/include
-- Looking for XOpenDisplay in opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libx11-1.8.10-dtf7uji6hrpgtmnjfljpwtx4bmdel3hm/lib/libX11.so;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxext-1.3.6-frmdae6dcynu4535k2ypww5jggnwpfxz/lib/libXext.so
-- Looking for XOpenDisplay in opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libx11-1.8.10-dtf7uji6hrpgtmnjfljpwtx4bmdel3hm/lib/libX11.so;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxext-1.3.6-frmdae6dcynu4535k2ypww5jggnwpfxz/lib/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- Found GLEW: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/glew-2.2.0-mo5jcq3naak3y6sdt5ltzkdhcpi4g4jz/lib/libGLEW.so
CMake Warning at VTK/Rendering/OpenGL2/CMakeLists.txt:451 (message):
  X11:Xcursor not found; custom cursors will be ignored.


-- Found LibXml2: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxml2-2.10.3-2cbjv2cwl4mm5bxmrwhiif6ghxrsf75w/lib/libxml2.so (found version "2.10.3")
-- Found HDF5: hdf5-shared (found version "1.14.5") found components: C HL
CMake Error at VTK/ThirdParty/hdf5/CMakeLists.txt:31 (message):
  An external MPI-aware HDF5 requires that VTK be built with MPI support as
  well.


-- Configuring incomplete, errors occurred!
==> Error: ProcessError: Command exited with status 1:
    'opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/cmake-3.30.5-h23fmzvmea77a4jg3dihcy4bz72jkk6b/bin/cmake' '-G' 'Ninja' -DCMAKE_PREFIX_PATH:STRING=opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/pugixml-1.10-2r72qtte56iwzvlnuqo4wigxpd5ofe64;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/protobuf-3.13.0-6p4hstndxpeeehliu53lksrkz4sgxjlp;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/proj-8.1.0-hx3d5v2h2zvgdvahke7vt3i33qbuz5bz;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/pegtl-2.8.3-5u5wvepk6t7fekt22fzkdf6gdbavecoa;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/nlohmann-json-3.11.3-ubipdqjwm3qag76vzl7n5wzv2x345fmh;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/netcdf-c-4.9.2-6euhkr4keva27azw2dxppn4rm7oix5d3;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libaec-1.0.6-gkzdtmodrwag7p4awsix33ekbiywffsr;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/hdf5-1.14.5-bxuqkul72wmaxuzdy36tvjk2ktqyciuq;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/openmpi-5.0.5-ykvt4t64hyztqgujekybpjnl4nfd6as5;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/pmix-5.0.3-4fdv6pbuworf2hsnkwp2bnfkwtemfmh5;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/numactl-2.0.18-ot33hns6raosys5gi2l5hcdvqk372vwq;/home/bkaindl/spack.5/opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libevent-2.1.12-s2ur47rbs6aygom4to6grygx4uxy5ats;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/c-blosc-1.21.5-hvaklx4p3nnqpng45rzbrwrbna7igkv2;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/snappy-1.1.10-fmwupli2sikp3jfbedobhp4a3iwtr4te;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/lz4-1.10.0-jjeqt2kye2avym36cxlke46s64zsfs6v;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libtiff-4.6.0-mdg3npn5i7hfzf5cqqywaunjzd2c46zu;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libtheora-1.1.1-joyidf5le4lbyztj36exsuebtjewsfbd;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libogg-1.3.5-66xpqstxcbgw2ncqgx2vzmqsuwtldugx;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libjpeg-turbo-3.0.3-tffgl7ijaojo5ec4ojr6lniafj7zqkfx;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/jsoncpp-1.9.5-467aebg2dqecutvtmsjz73fgjvzydl6y;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/glew-2.2.0-mo5jcq3naak3y6sdt5ltzkdhcpi4g4jz;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/glx-1.4-ldb6nnyqutqwoj4snm2ia7f7aokvcbp2;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/mesa-23.3.6-ccdk6jiz6lhbdfkyrapqqwwwzexcl2ks;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/xrandr-1.5.2-vottpkk73ljuar3hpvjqfalhlsvktejg;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxrandr-1.5.4-4eadn67orgvcgjuzrettm6gcg67tu2ux;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/randrproto-1.5.0-tlbxpvcglzmzmnow55rcgicl4bierdlu;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxrender-0.9.11-6lvxhpcdewfs4mafcm5j7kg4fpcxa7j7;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/renderproto-0.11.1-od2tcw3fbz2iu4pi7o6bnv4avuiyfxld;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/llvm-17.0.6-hywrlwv2bskuoca7fe6lfohwpctojfpf;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/swig-4.1.1-pbo27h3jn2t62tkxzwko2i4psb5ohnfs;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/pcre2-10.44-ohomucklfkbvybtvne7ui3glenebe7fx;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/ninja-1.12.1-wepbmyred3fgz62zbysduq6rxb4g646x;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/lua-5.3.6-xialgefhbgjzqhdhwdsuogpysjnlqm3q;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libedit-3.1-20240808-upfwxlljipi45okiyjpcxjbimzcn3tqb;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/hwloc-2.11.1-znd5x5ytxhvjepa3k2ordikjxk56ghdp;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libpciaccess-0.17-azx6duzhod5abnalp7keendpz4huhru4;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/binutils-2.43.1-4sodjgf6cc7uekitqepcc7dnmcwtaho3;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxt-1.3.0-ocjvjxavofrzwbihg4bpsqpagc3ofb2w;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libsm-1.2.4-o6cqylf6wpq5lxt664h555u2aqmv2gl7;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libice-1.1.1-4zsfoqozxzxie5grnkxwxr3onssb5ezk;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxext-1.3.6-frmdae6dcynu4535k2ypww5jggnwpfxz;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libx11-1.8.10-dtf7uji6hrpgtmnjfljpwtx4bmdel3hm;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/xtrans-1.5.0-k5kj5vhxaeeqafuhcwohtww7fduyfqmr;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/xextproto-7.3.0-vwtxuuuxps6dvrpf7cv7h7iurz76cquh;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxcb-1.17.0-rn4kx2ryk5qk5qqxey4qq6u6db7mfprs;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/util-linux-uuid-2.40.2-tyzqiklai55oisaoewn5ro5nv3wy7mcv;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/sqlite-3.46.0-i6xusas3hfs57j5p72at6gucksqd3gsw;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libffi-3.4.6-r6emmhbggetnzptutbkk7qffbsmezppz;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxdmcp-1.1.5-xxkwiljmjja7w7sp3kynvqyjz33rdiog;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxau-1.0.11-ldkt5yi3sok6eqydtuqzy6ubl2gk3bc4;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/xproto-7.0.31-xddze4r3uhsr6h27xijkzckrikpxoznf;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libpthread-stubs-0.5-doadflxu2oxvbyahsxtvvbfnw6tnn6z4;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/kbproto-1.0.7-jq43mwjkv35hsuxwa3rq7ahjncosez3u;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libunwind-1.6.2-u5yu7dhud7vpj3hyoty6x6357b7o3oif;/home/bkaindl/spack.5/opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/glproto-1.4.17-dqs2nfuwx3ioojm27hnqbn7su4sgirll;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/zstd-1.5.6-3oda7hrcm25fqymbhvrblqwlmpj7hwqh;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxml2-2.10.3-2cbjv2cwl4mm5bxmrwhiif6ghxrsf75w;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/xz-5.4.6-pwjjfgxqlhfn26tycgwy2iezdpzmt7br;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/freetype-2.13.3-hbq2y76onlaasodnlyex5iv745hhwpwl;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libpng-1.6.39-6ewrjni6gx3fuxbx7gy5b3giz7wtpwrx;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/expat-2.6.3-u2bucaey36mxkx6zbov2tk46okm3zsh2;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libbsd-0.12.2-calffvojasv62v5v6b4mplsycwbj67fs;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libmd-1.0.4-z4llf26xjzstptwwlm4efpzljny4hb47;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/eigen-3.4.0-bdamxaxlisqoxb2fuz4pb4vsxvx56gh5;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/double-conversion-3.3.0-4iuaeml6dsfeduksvrph3kaqgmcjxk5i;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/cli11-1.9.1-5r37fzco3pn3mdllq5a2swwiacxahl5v;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/cmake-3.30.5-h23fmzvmea77a4jg3dihcy4bz72jkk6b;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/curl-8.8.0-onnikjqv4vrfb4peuleyceuqwtv5sild;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/openssl-3.3.1-ry7k4rpxiacirqdesthhlc2c6mh5hhgy;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/zlib-ng-2.2.1-somd2q767lkqx6je2pz6am2gztdbovfe;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/readline-8.2-w6hn7avdihv3xjsswtwougplip4ulmgu;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/ncurses-6.5-5iauhqykhluyoehuhrwhvvuyvpud2765;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/nghttp2-1.63.0-kswzwok6iadc6xko2iwxljpij5y2gzd3;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/bzip2-1.0.8-geajalrno4mt3uers3ji7ytbkk6qvkez;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libiconv-1.17-jxe63kwxqplstnvcueiggoke5v2ezi6d;opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/gcc-runtime-13.1.0-25qo3ld7klulv5juopmjnk73qte743xz' -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON' '-DVTK_OPENGL_HAS_OSMESA:BOOL=OFF' '-DVTK_USE_X:BOOL=ON' '-DPARAVIEW_INSTALL_DEVELOPMENT_FILES:BOOL=OFF' '-DBUILD_TESTING:BOOL=OFF' '-DOpenGL_GL_PREFERENCE:STRING=LEGACY' '-DPARAVIEW_ENABLE_VISITBRIDGE:BOOL=OFF' '-DVISIT_BUILD_READER_Silo:BOOL=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_fast_float:BOOL=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_token:BOOL=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_verdict:BOOL=OFF' '-DVTK_MODULE_USE_EXTERNAL_ParaView_vtkcatalyst:BOOL=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_ioss:BOOL=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_exprtk:BOOL=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_fmt:BOOL=OFF' '-DPARAVIEW_BUILD_EDITION:STRING=CANONICAL' '-DPARAVIEW_USE_QT:BOOL=OFF' '-DPARAVIEW_BUILD_WITH_EXTERNAL=ON' '-DPARAVIEW_ENABLE_EXAMPLES:BOOL=OFF' '-DVTK_MODULE_USE_EXTERNAL_ParaView_cgns=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_gl2ps=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_libharu=OFF' '-DVTK_MODULE_USE_EXTERNAL_VTK_utf8=OFF' '-DPARAVIEW_ENABLE_PYTHON:BOOL=OFF' '-DPARAVIEW_BUILD_SHARED_LIBS:BOOL=OFF' '-DPARAVIEW_USE_CUDA:BOOL=OFF' '-DPARAVIEW_USE_HIP:BOOL=OFF' '-DPARAVIEW_ENABLE_RAYTRACING:BOOL=OFF' '-DVTK_ENABLE_OSPRAY:BOOL=OFF' '-DVTKOSPRAY_ENABLE_DENOISER:BOOL=OFF' '-DPARAVIEW_PLUGIN_ENABLE_CDIReader:BOOL=OFF' '-DPARAVIEW_PLUGIN_AUTOLOAD_CDIReader:BOOL=OFF' '.stage/spack-stage-paraview-5.13.1-ami44zd4mpzuklsdj2za5ln6oe4eguux/spack-src'

1 error found in build log:
     172    CMake Warning at VTK/Rendering/OpenGL2/CMakeLists.txt:451 (message):
     173      X11:Xcursor not found; custom cursors will be ignored.
     174
     175
     176    -- Found LibXml2: opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/libxml2-2.10.3-2cbjv2cwl4mm5bxmrwhiif6ghxrsf75w/lib/libxml2.so (found version "2.10.3")
     177    -- Found HDF5: hdf5-shared (found version "1.14.5") found components: C HL
  >> 178    CMake Error at VTK/ThirdParty/hdf5/CMakeLists.txt:31 (message):
     179      An external MPI-aware HDF5 requires that VTK be built with MPI support as
     180      well.
 
```

### General information

- [X] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [X] I have run `spack maintainers <name-of-the-package>` and **@mentioned** any maintainers
- [X] I have uploaded the build log and environment files
- [X] I have searched the issues of this repo and believe this is not a duplicate

--- a/share/spack/gitlab/cloud_pipelines/stacks/data-vis-sdk/spack.yaml
+++ b/share/spack/gitlab/cloud_pipelines/stacks/data-vis-sdk/spack.yaml
@@ -24,7 +24,7 @@ spack:
   definitions:
   - paraview_specs:
     - matrix:
-      - - paraview +raytracing
+      - - paraview +raytracing +adios2 +fides
       - - +qt ^[virtuals=gl] glx # GUI Support w/ GLX Rendering
         - ~qt ^[virtuals=gl] glx # GLX Rendering
         - ^[virtuals=gl] osmesa # OSMesa Rendering

--- a/var/spack/repos/builtin/packages/paraview/package.py
+++ b/var/spack/repos/builtin/packages/paraview/package.py
@@ -229,6 +229,7 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("tbb", when="+tbb")
 
     depends_on("mpi", when="+mpi")
+    conflicts("mpi", when="~mpi")
 
     depends_on("qt@:4", when="@:5.2.0+qt")
     depends_on("qt+sql", when="+qt")
@@ -609,11 +610,9 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
         else:
             cmake_args.append("-DPARAVIEW_ENABLE_PYTHON:BOOL=OFF")
 
-        cmake_args.append("-DPARAVIEW_USE_MPI:BOOL=%s" % variant_bool("+mpi"),
+        cmake_args.append("-DPARAVIEW_USE_MPI:BOOL=%s" % variant_bool("+mpi"))
         if "+mpi" in spec:
-            mpi_args = [
-                "-DMPIEXEC:FILEPATH=%s/bin/mpiexec" % spec["mpi"].prefix,
-            ]
+            mpi_args = ["-DMPIEXEC:FILEPATH=%s/bin/mpiexec" % spec["mpi"].prefix]
             if not sys.platform == "win32":
                 mpi_args.extend(
                     [

--- a/var/spack/repos/builtin/packages/paraview/package.py
+++ b/var/spack/repos/builtin/packages/paraview/package.py
@@ -609,9 +609,9 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
         else:
             cmake_args.append("-DPARAVIEW_ENABLE_PYTHON:BOOL=OFF")
 
+        cmake_args.append("-DPARAVIEW_USE_MPI:BOOL=%s" % variant_bool("+mpi"),
         if "+mpi" in spec:
             mpi_args = [
-                "-DPARAVIEW_USE_MPI:BOOL=ON",
                 "-DMPIEXEC:FILEPATH=%s/bin/mpiexec" % spec["mpi"].prefix,
             ]
             if not sys.platform == "win32":

--- a/var/spack/repos/builtin/packages/paraview/package.py
+++ b/var/spack/repos/builtin/packages/paraview/package.py
@@ -309,6 +309,8 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
     # and pre-5.9 is unable to handle that.
     depends_on("pugixml@:1.10", when="@:5.8")
     depends_on("pugixml", when="@5.9:")
+    # 5.13 uses 'remove_children': https://github.com/spack/spack/issues/47098
+    depends_on("pugixml@1.11:", when="@5.13:")
 
     # ParaView depends on cli11 due to changes in MR
     # https://gitlab.kitware.com/paraview/paraview/-/merge_requests/4951

